js/ui/codeView.js

```diff
@@ -193,7 +189,7 @@ function _flipButtonAroundRectCenter(props) {
 
 var WindowTrackingButton = new Lang.Class({
     Name: 'WindowTrackingButton',
-    Extends: GObject.Object,
+    Extends: St.Button,
```

Comment:
This commit makes me most happy. I was finishing a lengthy essay stating that this exact change should be done, but happilly am throwing it away now :slightly_smiling_face: